client/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "trabe",
+  "version": "1.0.0",
+  "description": "",
+  "main": "client.js",
+  "type": "module",
+  "scripts": {
+    "start": "nodemon exec node client.js"
```

```suggestion
    "start": "nodemon client.js"
```
It's just the same. Try to tuse the simplest version of the commands.